modules/Users/UserViewHelper.php

```diff
@@ -757,8 +757,6 @@ protected function setupEmailSettings() {
             $this->ss->assign('MAIL_SMTPPORT',$mail_smtpport);
             $this->ss->assign('MAIL_SMTPSSL',$mail_smtpssl);
         }
-        $this->ss->assign('HIDE_IF_CAN_USE_DEFAULT_OUTBOUND',$hide_if_can_use_default );
```

Comment:
Users can not use default outbound anymore but this calculation still there in file 
modules/Users/views/view.wizard.php at class ViewWizard::display()